vitest.config.ts

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vitest/config'
+
+export default defineConfig({
+  test: {
+    include: ['__test__/**.spec.ts'],
+    reporters: 'default',
+    environment: 'jsdom'
```

Comment:
I don't think we need `jsdom`, do we?
The default is `node`, so it should be OK.

Reply:
So to sum up, I think you can use the default config with:
```ts
export default defineConfig({
  test: {}
})
```